features/netsocket/ppp/ppp.c

```diff
@@ -927,8 +905,8 @@ void ppp_input(ppp_pcb *pcb, struct pbuf *pb) {
        */
       for (i = 0; (protp = protocols[i]) != NULL; ++i) {
         if (protp->protocol == protocol) {
-          pb = pbuf_coalesce(pb, PBUF_RAW);
-          (*protp->input)(pcb, (u8_t*)pb->payload, pb->len);
+          ///////// pb = pbuf_coalesce(pb, PBUF_RAW);
```

Comment:
Why commented out?

Reply:
Removed as not needed (always a single buffer on this stage).